src/main/java/org/wildfly/security/permission/PermissionUtil.java

```diff
@@ -435,13 +435,19 @@ public static Permission createPermission(final Class<? extends Permission> perm
             }
         }
         try {
-            if (twoArg != null) {
+            if (twoArg != null && name != null && actions != null) {
```

Comment:
Can we rely that IllegalArgumentException is thrown at least?
(If not, I think it will be better to close this issue without fix as not fixable then exception hacking...)